prac_08/unreliable_car_test.py

```diff
@@ -0,0 +1,10 @@
+from prac_08.unreliable_car import UnreliableCar
+
+
+def main():
+    unreliable_car = UnreliableCar("Prius 1", 100, 56)
+    unreliable_car.drive(40)
+    print(unreliable_car)
```

Comment:
Multiple tests are helpful when dealing with random situations. Consider using a loop.

Reply:
@lindsaymarkward Thanks for the feedback.